apps/admin/src/containers/BrokerCAD/Clients/ManualOverride.tsx

```diff
@@ -2,6 +2,7 @@ import React, { useState } from 'react';
 import { AnyAction } from 'redux';
 import { ActionType, storeTransition } from '@thecointech/broker-db';
 import { graph } from '@thecointech/tx-etransfer';
+import { etransfer } from '@thecointech/tx-deposit';
```

Comment:
possibly need better names than 'graph' for the exported transition graphs for our actions

Reply:
Is it linked to the "visual" graph?